swagger-java-wes-client/src/main/resources/OPENAPI_CREATION.md

```diff
@@ -0,0 +1,24 @@
+##How WES OpenApi 3.0 yaml was created from Swagger 2.0 yaml
+
+SwaggerHub was used to convert the WES 1.0.0 Swagger 2.0 yaml to OpenAPI 3.0 yaml
+
+####Steps to convert:
+1. Launch SwaggerHub https://app.swaggerhub.com/home
+2. Create an account and login to SwaggerHub
+3. Click on the 'plus' (**+**) icon on the left side of the page
+3. Select 'Import and Document API'
+4. In the Import API dialog paste the WES Swagger 2.0 URL: 
+https://raw.githubusercontent.com/ga4gh/workflow-execution-service-schemas/1.0.0/openapi/workflow_execution_service.swagger.yaml into the
+ 'Path or URL' edit box
+5. Click 'Import'
+..You should now be able to see the The WES Swagger 2.0 yaml in the editor
+6. Click on the green box with the down arrow at the top right of the editor
+7. Select 'Convert to OpenAPI 3.0'
+8. In the 'Convert to OpenAPI 3.0?' dialog click 'Convert & Update'
+9. Click the 'Export^' button on the uppper right hand area of the page
```

Comment:
The ^ should be pointed down, but calling it the Export dropdown menu might be sufficient.

Reply:
![image](https://user-images.githubusercontent.com/24548904/54209165-a9cb0480-44b3-11e9-9b25-c384ebb5f70f.png)

Reply:
fixed